sandboxes/tags.yaml

```diff
@@ -0,0 +1,31 @@
+#
+# Example yaml file containing input to the signadot cli
+# which defines a sandbox using tags.  Analytics covering
+# sandboxes with tags are available via the dashboard at
+# https://app.staging.signadot.com/analytics.
+#
+name: example-tags
```

Comment:
If we're using this just for the docs, and not an example involving CLI, I suggest not keeping it here just for the sake of hosting the file. Instead, this can be good place to include the tags:

https://docs.signadot.com/docs/cli#sandboxes

We can include the tags in the sample yaml there, and include a link to the tag docs page from there. Similarly, from the tags page, we can link to this yaml for a full yaml spec with tags.

Reply:
I very much disagree.  We can add more examples as actual yaml files that can be checked out and accessed via GitHub.  Then the file itself can be used as a reference, copied and edited in a script rather than digging into a web page for the same result.  For example, we keep a hotrod repo and kubectl apply can be used with it.  We can do similarly with the CLI.  Placing the file in a web page instead of in its natural usable form just makes it harder to use.   

We ourselves have all asked for example sandbox spec files:  myself, Arjun, Anirdudh at various times.  I think that is a strong indicator that it useful to provide them in examples. 

Over time, we could add examples with rollouts, resources, static endpoints, etc. and use them in scripts.

Reply:
I've made a proposal to do something similar here: https://www.notion.so/Brainstorming-5244505a4d9f40a4b9dd75ca2a8adb08?d=f395cbea7540426ab2d4cd9cfd887320#c689bcc565e04be194d845871b4087d6